spec/unit/recipes/default_spec.rb

```diff
@@ -11,3 +11,15 @@
     expect { :chef_run }.to_not raise_error
   end
 end
+
+describe 'default recipe on Ubuntu 14.04' do
+  let(:chef_run) do
+    ChefSpec::ServerRunner.new do |node|
+      node.automatic[:platform_family] = 'rhel'
```

Comment:
Going to update your existing spec and add another for ubuntu 16.04 based off this example https://github.com/sethvargo/chefspec/blob/master/examples/server/spec/node_spec.rb